vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -0,0 +1,426 @@
+---
+  - name: Define number of BMH's
+    set_fact:
+      NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
+
+  - name: Update maxSurge and maxUnavailable fields
+    shell: |
+           kubectl get machinedeployment -n {{NAMESPACE}} test1 -o json | 
```

Comment:
```suggestion
           kubectl get machinedeployment "{{ CLUSTER_NAME }}"  -n "{{ NAMESPACE }}"  -o json | 
```

Reply:
done

Reply:
test1 should be "{{ CLUSTER_NAME }}" ???